examples/graph3d/16_styling_surface.html

```diff
@@ -105,16 +118,34 @@
         />
         <label for="defaultStyle">Default Configuration: </label>
         <code>
-          <pre>surfaceColors: [
-  '#FF0000',
-  '#FFF000',
-  '#00FF00',
-  '#68E8FB',
+          <pre>colormap: [
   '#000FFF',
+  '#68E8FB',
+  '#00FF00',
+  '#FFF000',
+  '#FF0000',
 ]</pre>
         </code>
       </div>
 
+      <div>
+        <input
+          type="radio"
+          name="style"
+          id="functionStyle"
+          value="function"
+          onchange="drawVisualization();"
+        />
+        <label for="functionStyle">Function Configuration:</label>
+        <code>
+          <pre>colormap: function(x) {
+  const {r, g, b} = \
+    vis.util.HSVToRGB(1, x, 1);
```

Comment:
Don't use Vis Util. I know it's leaked into the wild and used all over the place, but it's intended to be private. We make breaking changes in it without any notice so it shouldn't appear in the examples.

Reply:
Ok, makes sense.